pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -39,6 +39,8 @@ type DatasourceInfo struct {
 }
 
 const CLOUDWATCH_TS_FORMAT = "2006-01-02 15:04:05.000"
+const LOG_IDENTIFIER_INTERNAL = "__log_grafana_internal__"
```

Comment:
Nitpick to be consistent I guess
```suggestion
const LOG_IDENTIFIER_INTERNAL = "__log__grafana_internal__"
```